__tests__/getOptions.test.js

```diff
@@ -30,3 +30,34 @@ describe('getOptions', () => {
 
   });
 });
+
+describe('replace <rootDir>', () => {
+  it('should replace <rootDir> in output path', () => {
+    const rootDir = 'testRootDir';
+    const outputPath = '<rootDir>/test/result/output.xml';
+
+    const newOutput = getOptions.replaceRootDirInOutput(rootDir, outputPath);
+
+    // Result of replaceRootDirInOutput will also contain the drive letter and path slashes (different format depending on OS).
+    // So instead assert that the rootDir is in the result and the placeholder (<rootDir>) is not.
+    expect(newOutput.indexOf(rootDir) !== 0).toBe(true);
```

Comment:
you can use `.not.toContain`, which gives better error messages. https://jestjs.io/docs/en/expect#tocontainitem

See https://repl.it/@SimenB/EnlightenedHarmfulCommand for example

Reply:
Thanks Simon. Latest commit contains changes to use the toContain matcher instead of indexOf.
Thanks

Reply:
It might be worth just adding a small sentence to the README in case people don't stumble on the issue/PR.